File: Sum of numbers from 0 to N/Sum of numbers from 0 to N.js
const SequenceSum = (function() {
  function SequenceSum() {}
  SequenceSum.showSequence = function(n) {
    if(n<0){
      return `${n}<0`
    }else if(n==0){
      return `0=0`
    }else{
      let result = []
      let tmp = 0;
      for(let i=0;i<=n;i++){
        result.push(String(i))
        tmp+=i
      }
      return result.join("+") + " =" +` ${tmp}`
    }
  };
  return SequenceSum;
})();

File: Sum of numbers from 0 to N/Sum of numbers from 0 to N.py
def show_sequence(n):
    if n < 0 :
        return str(n)+"<0"
    elif n==0:
        return "0=0"
    else:
        return "+".join(map(str,[i for i in range(0,n+1)]))+" = "+str(sum([i for i in range(0,n+1)]))
